packages/docusaurus-theme-classic/src/theme/DocPage/index.js

```diff
@@ -17,7 +17,7 @@ import styles from './styles.module.css';
 
 function DocPage(props) {
   const {route, docsMetadata, location} = props;
-  const {permalinkToSidebar, docsSidebars} = docsMetadata;
+  const {permalinkToSidebar, docsSidebars, sidebarCollapsible} = docsMetadata;
```

Comment:
It's quite complicated and maybe even redundant, no?

I mean, isn't it better to get the `sidebarCollapsible` value directly from `siteConfig`?

```js
const {siteConfig = {}} = useDocusaurusContext();
const = sidebarCollapsible = siteConfig.presets[""0""][1].docs.sidebarCollapsible; 
```

Yes, yes, it looks very terrible, but I want to find a solution easier.

Reply:
Use siteConfig.themeConfig for UI configuration/ client side stuff

metadata is something we generate in Node.js aka compile time

Reply:
@endiliey done :heavy_check_mark: